ztp/gitops-subscriptions/argocd/resource-hook-example/policygentemplates/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
```

Comment:
I'm not sure it makes sense to have a kustomization file here. These files are all committed to git where the argo pipeline is subscribed to them. We don't need a kustomization there.